lib/test/TestParser.js

```diff
@@ -200,31 +207,84 @@ module.exports = class TestParser {
         return assertion;
     }
 
-    parseStringAssertion(interaction, assertionString) {
+    // Parse a string to assertion object. 
+    // Our short syntax is not a valid in YAML. We look for operator inside the string
+    // Examples: 
+    //   Hi
+    //   exit
+    //   LaunchRequest != hi
+    //   cardTitle != hi
+    //   cardTitle != - Value1 - Value2 - Value2
+    // if isUtterIncluded is set to true, the first part of the string will be consider as utter
```

Comment:
This should be labeled "isUtteranceIncluded" - isUtterIncluded does not make sense